srcpkgs/olive/template

```diff
@@ -0,0 +1,21 @@
+# Template file for 'olive'
+pkgname=olive
+version=20181130
```

Comment:
Can we get a `0.0.0.` prepended to this, so we can leave them room to do semver once they get out of alpha?